lib/internal/http2/compat.js

```diff
@@ -568,10 +568,8 @@ class Http2ServerResponse extends Stream {
     if (this[kStream].headersSent)
       throw new ERR_HTTP2_HEADERS_SENT();
 
-    // If the stream is destroyed, we return false,
-    // like require('http').
```

Comment:
I couldn't find the code which this refers to. The `writeHead` method in the `http` module appears to have no explicit returns.